interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+<?php
+//by Art Eaton
+include_once("../../globals.php");
+include_once("$srcdir/api.inc");
+formHeader("Form: button_ros");
+?>
+
+<form method=post action="<?echo $rootdir;?>/forms/button_ros/save.php?mode=new" name="my_form">
+
+ 
+<?php
+$tempquery = "SELECT date FROM form_encounter WHERE encounter = $encounter";
+ $datequery = sqlStatement($tempquery);
+$encdate = SqlFetchArray($datequery);
+
+$old_date = strtotime($encdate['date']);
+$todays_date = strtotime(date("Ymd"));
+if($old_date+(86400) < $todays_date) {
+echo xlt('It is not permitted to enter a Review of Systems form in an encounter that is dated earlier than 3 days prior to today\'s date.');
+} else {
+?>
+<a href="javascript:top.restoreSession();document.my_form.submit();" class="link_submit">[<?php echo xlt('Save');?>]</a>
+<a href="<?php echo $GLOBALS['form_exit_url']; ?>" class="link_submit"
+ onclick="top.restoreSession()">[<?php echo xlt('Don\'t Save');?>]</a>
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation